data/cy/census_individual_gb_wls.json

```diff
@@ -7438,7 +7438,7 @@
                                         "guidance": {
                                             "content": [
                                                 {
-                                                    "title": "Current serving members should select \u201cNo\u201d"
+                                                    "title": "Aelodau sy\u2019n gwasanaethu ar hyn o bryd, dewiswch \u2018Nac ydy\u2019 yn unig"
```

Comment:
```suggestion
                                                    "title": "Aelodau sy\u2019n gwasanaethu ar hyn o bryd, dewiswch \u2018Nac ydw\u2019"
```

Reply:
Won't 'Nac ydw' here need to changed back to 'Nac ydy' if answering on behalf of someone else?

Reply:
This variant should only be shows when they are answer for themselves. If you look a bit further down, there's a variant for when the user is answering on behalf of someone else

Reply:
Yes, sorry just noticed that and deleted my comment

Reply:
So "Nac ydw" would be "I do not" and "Nac ydy" for "They do not"